src/pages/references/node-configuration.md

```diff
@@ -0,0 +1,489 @@
+
+```bash
+stacks-node mocknet
+```
+
+### krypton (deprecated)
+
+Start a node that will join and stream blocks from the public krypton testnet, powered by Blockstack via (Proof of Transfer).
+
+Example:
+
+```bash
+stacks-node krypton
+```
+
+### xenon
+
+Start a node that will join and stream blocks from the public xenon testnet, decentralized.
+
+Example:
+
+```bash
+stacks-node xenon
+```
+
+### start
+
+Start a node with a config of your own. Can be used for joining a network, starting new chain, etc.
+
+#### Arguments
+
+**--config**: relative or absolute path to the TOML config file. Required.
+
+Example:
+
+```bash
+stacks-node start --config=/path/to/config.toml
+```
+
+See [Configuration File Options](#configuration-file-options) for more information.
+
+#### version
+
+Displays information about the current version and the release cycle.
+
+Example:
+
+```bash
+stacks-node version
+```
+
+#### help
+
+Displays a help message.
+
+Example:
+
+```bash
+stacks-node help
+```
+
+## Configuration File Options
+
+The TOML configuration file has multiple sections under which an option may be placed.
+
+To see a list of example configurations, [please see this page](https://github.com/blockstack/stacks-blockchain/tree/master/testnet/stacks-node/conf).
+
+### Section: node
+
+Contains various configuration options pertaining to the stacks-node.
+
+Example:
+
+```toml
+[node]
+rpc_bind = "0.0.0.0:20443"
+p2p_bind = "0.0.0.0:20444"
+bootstrap_node = "048dd4f26101715853533dee005f0915375854fd5be73405f679c1917a5d4d16aaaf3c4c0d7a9c132a36b8c5fe1287f07dad8c910174d789eb24bdfb5ae26f5f27@testnet-miner.blockstack.org:20444"
+# Enter your private key here!
+seed = "replace-with-your-private-key"
+miner = true
+```
+
+#### working_dir (optional)
+
+Absolute path to the directory which the stacks-node will use for storing various data.
+
+Example:
+
+```toml
+working_dir = "/root/stacks-node"
+```
+
+#### rpc_bind
+
+Address and port stacks-node should bind to for RPC connections.
+
+Example:
+
+```toml
+rpc_bind = "0.0.0.0:20443"
+```
+
+#### p2p_bind
+
+Address and port stacks-node should bind to for P2P connections.
+
+Example:
+
+```toml
+p2p_bind = "0.0.0.0:20444"
+```
+
+#### data_url (optional)
+
+Address and port from which the stacks-node will be receiving incoming rpc connections.
+
+Example:
+
+```toml
+data_url = "1.2.3.4:20443"
+```
+
+#### p2p_address (optional)
+
+Address and port from which the stacks-node will be receiving incoming p2p connections.
+
+Example:
+
+```toml
+p2p_address = "1.2.3.4:20444"
+```
+
+#### bootstrap_node (optional)
+
+Public key, address, and port stacks-node should use to pull transaction data from when starting.
+
+Example:
+
+```toml
+bootstrap_node = "048dd4f26101715853533dee005f0915375854fd5be73405f679c1917a5d4d16aaaf3c4c0d7a9c132a36b8c5fe1287f07dad8c910174d789eb24bdfb5ae26f5f27@krypton.blockstack.org:20444"
+```
+
+#### wait_time_for_microblocks (optional)
+
+Time to wait (in milliseconds) for microblocks.
```

Comment:
Not clear to me what it means to wait for microblocks.

Reply:
@kantai or @lgalabru any insight on this one?

Reply:
This is the amount of time that a node will wait before trying to mine a block, after catching up to the anchored chain tip. This gives the current leader time to broadcast microblocks that will get included in that mined block.